Azure/sonic-buildimage
# [sonic-mgmt]: Upgrade scapy

Signed-off-by: Ze Gan <ganze718@gmail.com>



#### Why I did it
The version of scapy in apt source is out-of-date to MACsec.

#### How I did it
Upgrade scapy by pip

#### How to verify it
Check the version of scapy that should be 2.4.5
```
root@5e327a6ebcb5:/# python
Python 2.7.17 (default, Feb 27 2021, 15:10:58)
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import scapy
>>> scapy.__version__
'2.4.5'

```

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
